gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sat May 20 14:19:53 MST 2017
+#Thu Jun 01 10:45:08 MST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-rc-1-bin.zip
```

Comment:
Mild concerns. I anticipate an RC 2 before GA, so GA is still probably a month out. I don't anticipate any new breaking changes or major new features to be added before then.

My primary concerns are that:
* uPortal 5 could be cut without Gradle 4 and the possibility of being trapped on Gradle 3 until the next major uPortal release.
* During the transition to Gradle some custom tasks are being created that could be handled by new tasks bundled in Gradle 4. Those will take time and energy to rework.